src/app/api/stripe/membership/manage/route.ts

```diff
@@ -1,9 +1,9 @@
-import {type NextApiRequest, NextApiResponse} from "next";
-import {stripe} from "@/services/stripe";
-import {env} from "@/env";
-import {NextResponse} from "next/server";
+import { type NextApiRequest, NextApiResponse } from "next";
```

Comment:
Do we have any linter settings to edit in order to automatically fix these?

Reply:
Yes I am using Prettier

Reply:
#31 needs to be implemented to avoid lint errors leaking in build phase